apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx

```diff
@@ -80,10 +80,8 @@ export const InboundPassiveStsForm: FunctionComponent<InboundPassiveStsFormProps
 
     const { t } = useTranslation();
 
-    const [ isPEMSelected, setPEMSelected ] = useState<boolean>(false);
     const [ showCertificateModal, setShowCertificateModal ] = useState<boolean>(false);
-    const [ PEMValue, setPEMValue ] = useState<string>(undefined);
-    const [ certificateDisplay, setCertificateDisplay ] = useState<DisplayCertificate>(null);
+    const [ certificateDisplay ] = useState<DisplayCertificate>(null);
```

Comment:
I'm not sure. These changes were done due to ESLint errors. The `setCertificateDisplay` was not used anywhere in the page, hence why it was removed.

Reply:
Then, most probably `certificateDisplay` too shouldn't be necessary. Since this variable's value is null and it wouldn't be updated later, we can remove this state variable and use `null` directly wherever this variable is used.

Reply:
Updated https://github.com/wso2/identity-apps/pull/4518/commits/51b130fc62dde0229feff37fe7bd25dac7954c59